tools/ccm-service-kit/composer/dags/dependencies/variables/variables.json

```diff
@@ -0,0 +1,13 @@
+{
+    "project_id" : "ccm-dev-sandbox",
+    "ccm_bucket" : "ccm-zgnzp0",
+    "prefix_azure_lz" : "azure-lz",
+    "prefix_aws_lz" : "aws-lz",
+    "prefix_hist_azure" : "azure-historical",
+    "prefix_hist_aws" : "aws-historical",
+    "sts_id_azure" : "transferJobs/13017487796363048406",
+    "sts_id_aws" : "transferJobs/16024774301148900524",
+    "gcp_billing_table" : "gcp_billing",
+    "ccm_staging_dataset" : "cloud_cost_staging",
+    "ccm_final_dataset" : "cloud_cost_final"
```

Comment:
Please restrain from using the actual name of the resources and use place holders. For example instead of "ccm-dev-sandbox" use <project-name>